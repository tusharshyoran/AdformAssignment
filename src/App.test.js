import React from "react";
import { Provider } from "react-redux";
import store from "./redux/store";
import App from "./App";
import renderer from "react-test-renderer";

describe("App", () => {
  it("should render correctly", () => {
    const component = renderer
      .create(
        <Provider store={store}>
          <App />
        </Provider>
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
