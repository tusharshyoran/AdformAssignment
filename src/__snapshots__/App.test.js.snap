// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="mt-4"
  >
    <div
      className="flex-container"
    >
      <div
        className="react-datepicker-wrapper"
      >
        <div
          className="react-datepicker__input-container"
        >
          <input
            className=""
            disabled={false}
            onBlur={[Function]}
            onChange={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            placeholder="Start Date"
            readOnly={false}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className=""
      >
        <div
          className="react-datepicker-wrapper"
        >
          <div
            className="react-datepicker__input-container"
          >
            <input
              className=""
              disabled={false}
              onBlur={[Function]}
              onChange={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              placeholder="End Date"
              readOnly={false}
              type="text"
              value=""
            />
          </div>
        </div>
      </div>
      <div
        className="search-right"
      >
        <input
          onChange={[Function]}
          placeholder="Search by name"
          type="text"
          value=""
        />
      </div>
    </div>
    <div
      className="mt-4"
    >
      <div>
        <table
          className="table"
        >
          <thead>
            <tr>
              <th
                scope="col"
              >
                Name
              </th>
              <th
                scope="col"
              >
                User Name
              </th>
              <th
                scope="col"
              >
                Start Date
              </th>
              <th
                scope="col"
              >
                End Date
              </th>
              <th
                scope="col"
              >
                Active
              </th>
              <th
                scope="col"
              >
                Budget
              </th>
            </tr>
          </thead>
        </table>
      </div>
    </div>
  </div>
</div>
`;
